GrootCI.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
we want this to work upon asserting a push.

Reply:
why?

Reply:
because there is no point in checking a piece of code that may or may not be inserted to the master branch, we want anything that is requesting to be pushed to the master branch to be approved/disapproved depending on if it maintains the status quo.